internal/oci/runtime_vm.go

```diff
@@ -680,6 +748,10 @@ func (r *runtimeVM) updateContainerStatus(ctx context.Context, c *Container) err
 		return errdefs.ErrNotFound
 	}
 
+	if err = r.restoreContainerIO(ctx, c, response); err != nil {
```

Comment:
in the case where we haven't restarted, does this get called? should it?

Reply:
yes, if container io has already existed in ctrs map, `restoreContainerIO` won't do more operations.